src/main/resources/config/application-prod.yml

```diff
@@ -22,10 +22,10 @@ spring:
     datasource:
         type: com.zaxxer.hikari.HikariDataSource
         driverClassName: org.postgresql.Driver
-        url: jdbc:postgresql://localhost:5432/managementportal
+        url: jdbc:postgresql://192.168.56.101:5432/mp
 #        name: managementportal
-        username: <username>
-        password: <password>
+        username: mp
+        password: secret
```

Comment:
Is this the actual database password? If so, it's not a good idea to put it on GitHub. If this is just a default value, I would prefer it to be something like CHANGE_ME.

Reply:
Indeed, i was testing the production profile locally and accidentally committed the changes. Changes to this file have been reverted.